DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -132,7 +132,9 @@
     <![CDATA[superClusterRelinked_.reset();]]>
   </ioread>
 
-  <class name="pat::Jet"  ClassVersion="14">
+  <class name="pat::Jet"  ClassVersion="16">
+   <version ClassVersion="16" checksum="727883729"/>
+   <version ClassVersion="15" checksum="3059757499"/>
```

Comment:
I see. Since 15 was a temporary version appearing only on your developments branch, why not delete the version 15 checksum and rename 16 to 15?